### PR TITLE
Ember data 2 2 blog

### PR DESCRIPTION
**Pull request by bmac**
cc @fivetanley @igorT @wecc

### CODE REVIEW

File: source/blog/2015-11-17-ember-data-2-2-released.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Ember Data 2.2 and 2.3 Beta Released
+author: Brendan McLoughlin
+tags: Releases
+---
+
+Ember Data 2.1, a minor version release of Ember Data with bug fixes
```
Comment (rwjblue) [inline source/blog/2015-11-17-ember-data-2-2-released.md]: 2.2

File: source/blog/2015-11-17-ember-data-2-2-released.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+only, is released today. This release represents the work of over 21
+direct contributors, and over 140 commits.
+
+Ember Data 2.2 beta.1, the branch of Ember Data that will be released
```
Comment (rwjblue) [inline source/blog/2015-11-17-ember-data-2-2-released.md]: 2.3 beta.1

File: source/blog/2015-11-17-ember-data-2-2-released.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+as stable in roughly six weeks, is also being released today.
+
+### Changes in Ember Data 2.2
+
+There are no new features introduced in Ember Data 2.1. Instead the
```
Comment (rwjblue) [inline source/blog/2015-11-17-ember-data-2-2-released.md]: 2.2

Comment (mixonic) [inline source/blog/2015-11-17-ember-data-2-2-released.md]: 2.2 :-)

**Pull request merged**